Configuration/CharacterList.cs

```diff
@@ -9,6 +9,11 @@ namespace Sequencer.Configuration
     [XmlRoot(ElementName = "Words")]
     public class CharacterList : CustomSerializationBaseList<char>
     {
+        public CharacterList()
+        {
+            BackingList = new List<char>();

```

Comment:
This is already done in the base class, look at `CustomSerializationBaseList<T>.ctor`